crates/ide_assists/src/handlers/expand_glob_import.rs

```diff
@@ -199,9 +199,8 @@ fn find_refs_in_mod(
 fn is_mod_visible_from(ctx: &AssistContext, module: Module, from: Module) -> bool {
```

Comment:
this should probably really be operation on paths (is path `foo::bar::baz` visible from `from`?)

Reply:
Hm, I don't think it can? You need to know `bar`'s visibility for that, and that's impossible to do with just the path?

Reply:
well, you'd need to resolve each segment as you go of course. Maybe the right API would be a version of `sema.resolve_path` that checks visibility at the same time?